Assets/00_Content/Scripts/Rounds/ScalingData.cs

```diff
@@ -33,6 +33,9 @@ public class ScalingData : ScriptableObject {
 		[SerializeField, Min(0), Tooltip("Should be higher than InitialMoodDeclineMultiplier")]
 		private float maximumMoodDeclineMultiplier = 2f;
 
+		[Min(0)]
+		public float tableHealth = 100f;
```

Comment:
So, table gets its health based on the difficulty, is that right? I'm wondering if we want to make table health have something to do with tavern health? So that they are calculated based on each other, or if both are maybe set by the same ScalingData?
My thought is that we might want to make it so that tavern health is equal to the health of all tables taken together? Or set tavern health to slightly less than that maybe?

Reply:
Yes, good idea.  I do see this as a future feature though.